lib/prometheus_exporter/client.rb

```diff
@@ -57,6 +57,12 @@ def initialize(host: 'localhost', port: PrometheusExporter::DEFAULT_PORT, max_qu
     @thread_sleep = thread_sleep
 
     @json_serializer = json_serializer == :oj ? PrometheusExporter::OjCompat : JSON
+
+    @custom_labels = custom_labels
```

Comment:
A tiny bit wasteful, let's just let custom_labels be nil and only merge if it is not nil.

Reply:
sure